plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -406,9 +406,15 @@ public static boolean canEnforceColumnConstraintInSpecs(
             IcebergColumnHandle columnHandle,
             Domain domain)
     {
-        return table.specs().values().stream()
```

Comment:
`Due to empty_stream.allMatch always returns True` -> use a sentence 

The constraint check was previously misbehaving when dealing with empty tables
due to `empty_stream.allMatch(...)` always returns `True`.